wordpress-stubs.php

```diff
@@ -130182,6 +130191,7 @@ function get_tax_sql($tax_query, $primary_table, $primary_id_column)
      * @param string             $filter   Optional. How to sanitize term fields. Default 'raw'.
      * @return WP_Term|array|WP_Error|null WP_Term instance (or array) on success, depending on the `$output` value.
      *                                     WP_Error if `$taxonomy` does not exist. Null for miscellaneous failure.
+     * @phpstan-param 'OBJECT'|'ARRAY_A'|'ARRAY_N' $output
      * @phpstan-return ($output is 'ARRAY_A' ? array<string, string|int>|\WP_Error|null : ($output is 'ARRAY_N' ? list<string|int>|\WP_Error|null : \WP_Term|\WP_Error|null))
      */
     function get_term($term, $taxonomy = '', $output = \OBJECT, $filter = 'raw')
```

Comment:
core says: constant
phpstan says: string
viktor says: WordPress
```php
define( 'OBJECT', 'OBJECT' );
```

Reply:
To be fair, core also says string but passes the default string as a constant. But yeah, `define( 'OBJECT', 'OBJECT' );` is weird.